Toolkit concept docs XAML: batch 1 (#554)
Esri/arcgis-maps-sdk-dotnet-toolkit: docs/floor-filter.md

## Usage

### .NET MAUI:

```xml
<Grid>
    <esri:MapView x:Name="mapView" />
    <toolkit:FloorFilter GeoView="{x:Reference mapView}" />
</Grid>
```

### UWP/WinUI:

```xml
<Grid>
    <esri:MapView x:Name="mapView" />
    <esri:FloorFilter GeoView="{Binding ElementName=mapView}" />
</Grid>
```

### WPF:

> The usage in WPF is identical to UWP/WinUI minus one important distinction. The `FloorFilter` should be accessed with the same prefix as the `GeoView`. 

Reviewer: ```suggestion
The usage in WPF is identical to UWP/WinUI with one important distinction: `FloorFilter` is accessed with the same namespace prefix as `GeoView`.
```

```xml

> xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013"

Reviewer 2: I liked it better when this was declared on a root-element of the XAML sample, rather than a sub-note
